gitpod-io/website
# feat: add C# .NET template to quickstart page

## Description
Added C# .NET quickstart page.

## Related Issue(s)
fixes [Internal Discussion](https://gitpod.slack.com/archives/C01KPEPNBD0/p1660093823993819)

## How to test
- Open Deployed Preview link from the Netlify bot
- Go to `/docs/quickstart` (you will see `C# .NET` there)
- Go to `/docs/quickstart/dotnet`

## Release Notes

```release-note
added C# Dotnet Quickstart page
```

## Documentation
NO


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2555"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
